### ISSUE TITLE
[Settings> Actions]: At 200% resize Text, Shortcut keys (Ctrl+numpad_minus & Ctrl+numpad_plus) are not fully visible inside Actions page.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#12006]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Pre-requisites:**
1. Navigate to Settings>System> Dispaly.
2. Then Set Display Rsolution to 1280 x 768 and Scale at 100%.
3. Then Navigate to Settings>Accessibility>Text Size
4. Apply 200% text resize. 

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Then navigate inside Actions page and observe the issue.

**User Experience:**
Low vision users UX will not able good if they are not able to see the full text.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/resize-text

**Attachments:**
[At 200 resize Text, Shortcut keys are not fully visible inside Actions page..zip](https://github.com/microsoft/terminal/files/7750395/At.200.resize.Text.Shortcut.keys.are.not.fully.visible.inside.Actions.page.zip)

### Expected Behavior

At 200% resize Text, Shortcut keys (Ctrl+numpad_minus & Ctrl+numpad_plus) should be fully visible inside Actions page.

### Actual Behavior

At 200% resize Text, Shortcut keys (Ctrl+numpad_minus & Ctrl+numpad_plus) are not fully visible inside Actions page. User is only able to see 'Ctrl+numpad_mi' & 'Ctrl+numpad_pl'.

**Observation:** User can see full shortcut text in Shortcut editable mode.
